api/app/main.py

```diff
@@ -86,8 +88,34 @@
 @app.get('/health')
 async def get_health():
     """ A simple endpoint for Openshift Healthchecks """
-    LOGGER.info('/health')
-    return {"message": "Healthy as ever"}
+    try:
+        parts = [
+            config.get('PATHFINDER_BASE_URI'),
+            '/apis/apps/v1beta1/namespaces/',
+            config.get('PROJECT_NAMESPACE'),
+            'statefulsets',
+            config.get('PATRONI_HEALTH_SUFFIX')
+        ]
+        url = reduce(os.path.join, parts)
```

Comment:
`url = "/".join(map(lambda part: part.rstrip('/'), parts))`

Reply:
Taken from: https://stackoverflow.com/questions/1793261/how-to-join-components-of-a-path-when-you-are-constructing-a-url-in-python